2023/AzureSingapore/data.json

```diff
@@ -0,0 +1,28 @@
+{
+  "ValidMaintainers": {
+    "Description": "GitHub users who maintain this folder. PR for changing this folder will only be accepted from these users.",
+    "GitHubHandles": [
+      "nileshgule"
+    ]
+  },
+  "CommunityInfo": {
+    "Name": "Azure Singapore",
+    "Country": "Singapore",
+    "Url": "https://www.meetup.com/en-AU/mssgug/events/291424203",
+    "SessionizeApiId": "{Your sessionize event Id}",
+    "Description_SessionizeApiId": "https://sessionize.com/api/v2/10616/view/Sessions",
+    "SessionizeEventName": "https://sessionize.com/global-azure-bootcamp-singapore-2023/",
+    "Logo": {
+      "Description": "Azure Singapore Community Logo",
+      "File": "Azure-Singapore.png"
+    },
+    "Locations": [
+      {
+        "Name": "Global Azure Bootcamp Singapore",
+        "Address": "{Optional address of the location}",
```

Comment:
I see the latitude and longitude are provided, is it correct that addresses are not set? It's optional but just making sure.

Reply:
@NileshGule is it the case that you don't know the street address of your event just yet?

Reply:
@noopman Yes we are still finalizing the venue. In the past we used to conduct the Singapore Global Azure Bootcamp in the Microsoft office. This year we might have some logistic issues, so we are still in discussion with folks from Microsoft Singapore.